spec/lib/mobile_detection_spec.rb

```diff
@@ -36,11 +36,13 @@
 
   it "detects modern browsers correctly" do
     modern_user_agents.each do |agent|
-      expect(MobileDetection.modern_mobile_device?(agent)).to eq(true)
+      expect(MobileDetection.modern_mobile_device?(agent)).to eq(false),
+      "Failed User Agent: '#{agent}'"
     end
 
     old_user_agents.each do |agent|
-      expect(MobileDetection.modern_mobile_device?(agent)).to eq(false)
+      expect(MobileDetection.modern_mobile_device?(agent)).to eq(false),
+      "Failed User Agent: '#{agent}'"
```

Comment:
The lack of indentation here is strange and a bit misleading.

Reply:
Added parenthesis